apps/web/src/components/LanguageSelector.tsx

```diff
@@ -0,0 +1,83 @@
+import React, { Fragment } from 'react';
+import { Menu, Transition } from '@headlessui/react';
+import { ChevronDownIcon } from '@heroicons/react/solid';
+import classNames from 'classnames';
+import { useTranslation } from 'react-i18next';
+import { useMediaQuery } from 'react-responsive';
+
+import globe from '../images/globe.svg';
+
+const languages = {
+  en: 'English',
+  fr: 'Français',
```

Comment:
Lets keep it iterative. Just keep english for now. we can add the other languages later when we implement them